src/commands/cmd_delete.c

```diff
@@ -71,6 +71,12 @@ void _MGraph_Delete(void *args) {
         // Log error!
     }
 
+    // Remove GraphContext from keyspace.
+    key = GraphContext_Key(ctx, graphIDStr);
```

Comment:
I'm missing indices deletion

Reply:
Indices are directly owned by the GraphContext (they don't have separate keys), and as such get freed during the GraphContext_Free callback the deletion here triggers.